src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1696,6 +1788,173 @@ export class PatientViewPageStore {
         []
     );
 
+    readonly oncoKbDataForOncoprint = remoteData<IOncoKbData | Error>(
+        {
+            await: () => [this.mutationData, this.oncoKbAnnotatedGenes],
+            invoke: async () =>
+                fetchOncoKbDataForOncoprint(
+                    this.oncoKbAnnotatedGenes,
+                    this.mutationData
+                ),
+            onError: (err: Error) => {
+                // fail silently, leave the error handling responsibility to the data consumer
+            },
+        },
+        ONCOKB_DEFAULT
+    );
+
+    readonly cnaOncoKbDataForOncoprint = remoteData<IOncoKbData | Error>(
+        {
+            await: () => [
+                this.clinicalDataForSamples,
```

Comment:
how come we await things that aren't used in invoke?

Reply:
good catch. this might be some leftover code.